src/main/scala/chisel/compatibility.scala

```diff
@@ -0,0 +1,139 @@
+// See LICENSE for license details.
+
+// Allows legacy users to continue using Chisel (capital C) package name while
+// moving to the more standard package naming convention chisel (lowercase c).
+
+package object Chisel {
```

Comment:
They were ported to chisel2.
On May 20, 2016 7:25 PM, "Richard Lin" <notifications@github.com> wrote:

> In src/main/scala/chisel/compatibility.scala
> <https://github.com/ucb-bar/chisel3/pull/197#discussion_r64126635>:
>
> > @@ -0,0 +1,139 @@
> > +// See LICENSE for license details.
> > +
> > +// Allows legacy users to continue using Chisel (capital C) package name while
> > +// moving to the more standard package naming convention chisel (lowercase c).
> > +
> > +package object Chisel {
>
> Aren't those new chisel3 features? I think it would be nice to keep the
> new chisel3 features in chisel, otherwise we're going to end up
> maintaining this forever.
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/ucb-bar/chisel3/pull/197/files/d8381250d2f18d99edfd527a1353d5fa52fad10a#r64126635>
>